README.md

```diff
@@ -54,3 +54,58 @@ spark.optimizeSpatialAuto(sourceTable, outputTable, outputLocation)
 // optimize with the user defined block size
 spark.optimizeSpatial(sourceTable, outputTable, outputLocation, blockSize = 20097000)
 ```
+
+### Registering Optimizations on Databricks
+
+> This section and approach are based on [docs]](https://sedona.apache.org/setup/databricks/)
+> from Apache Sedona.
+
+Installing and registering spark optimizations can be tricky. As is often the case when
+working with distributed systems, there are some fundamental sequencing issues that are
+important to understand. Roughly, the startup of a databricks cluster looks something like
+this:
```

Comment:
Hmmm, mb not to have all of these exaplanations here, just to have the TLDR; first and somewhere below explanations; what do you think?

i.e. the following plan:

1. create the init script
2. change cluster configuration
3. explain why we need all the steps above, i.e.
    * configuration change to enable optimizations on the cluster startup
    * init script to ensure that optimizations exist in the cluster classpath on the spark configuration initialization step